Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 - Fixed memory allocation on every frame when using UIDocument without EventSystem. [ISXB-953](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-953)
 - Fixed Action Maps name edition which could be inconsistent in Input Action Editor UI.
+- Fixed touch pointers being released twice causing an index out of bounds error.
+
+### Added
```

Comment:
Where did these "### Added .... Hinge Angle" CHANGELOG.md lines come from?

Reply:
It comes from merging develop branch into this one in commit 6715e263e30b1971ca69a8334aa095df693d88c8

Reply:
No it's a duplicate of what was included in 1.11 release notes: https://github.com/Unity-Technologies/InputSystem/blob/develop/Packages/com.unity.inputsystem/CHANGELOG.md, double check against develop